Shopify/vscode-ruby-lsp
# Stage rl/standard debug sockets

### Motivation

Closes [#832](https://github.com/Shopify/vscode-ruby-lsp/issues/832)

Simplify debugging rails app and running tests. Use standard debug sockets provided by rdbg as much as possible.

### Implementation

Attach configuration - request sockets by running `bundle exec rdbg --util=list-socks`
Running tests - extract socket info from rdbg output
Updated Readme.md to reflect this

### Automated Tests

No tests added. If required, let me know

### Manual Tests

Install `debug` gem. Verify by running `bundle exec rdbg -v`
Run your app with rdbg - e.g. `bundle exec rdbg -O -n -c -- bin/rails server -p 3000`
You should be able to use vscode debugger attach configuration to connect to your running app
